xp/csp_run_parallel.sh

```diff
@@ -0,0 +1,44 @@
+# run the experiments in parallel, using as many CPU cores as available
+# supposed to be called at the root of the project
+
+# ------ parameters for the run -------
+declare -a BanditType=("ucbNew" "epsilonGreedyNew")
```

Comment:
Quick note: I think we can rename `ucb` and `ucbNew` to `ucbOld` and `ucb`, respectively. Probably better to do this in a separate PR since it requires changing the names in the code too of course